pkg/cluster/nodes/create.go

```diff
@@ -80,7 +84,7 @@ func CreateControlPlaneNode(name, image, clusterLabel, listenAddress string, por
 
 // CreateExternalLoadBalancerNode creates an external loab balancer node
 // and gets ready for exposing the the API server and the load balancer admin console
-func CreateExternalLoadBalancerNode(name, image, clusterLabel, listenAddress string, port int32) (node *Node, err error) {
+func CreateExternalLoadBalancerNode(name, image, clusterLabel, listenAddress string, port int32, portMappings []cri.PortMapping) (node *Node, err error) {
```

Comment:
same comment as above, is there an use case for port mapping in the load balancer node?

Reply:
I added the `ExtraPortMapping` capability to all roles but I think I agree with you, it doesn't make too much sense having this feature in load balancer nodes, I'll revert these changes here.